lib/MusicBrainz/Server/Filters.pm

```diff
@@ -47,7 +47,9 @@ sub format_wikitext
     return decode(
         'utf-8',
         Text::WikiFormat::format(
-            encode('utf-8' => $text), {}, {
+            encode('utf-8' => $text), {
+                paragraph => [ '<bdi><p>', '</p></bdi>'  ],
```

Comment:
This works because (some?) browsers don't care, but `<p>` is *not* allowed inside `<bdi>`. Would the other way work here?